ui/src/tempVars/components/TemplateVariableEditor.tsx

```diff
@@ -202,16 +213,46 @@ class TemplateVariableEditor extends PureComponent<Props, State> {
     return component
   }
 
-  private handleUpdateTemplate = (nextTemplate: Template): void => {
-    this.setState({nextTemplate})
+  private handleUpdateSelectedTemplateValue = (
+    selected: TemplateValue
+  ): void => {
+    const {
+      nextTemplate,
+      nextTemplate: {values},
+    } = this.state
+
+    const nextValues = values.map(v => {
+      if (v.value === selected.value) {
+        return {...v, selected: true}
+      } else {
+        return {...v, selected: false}
+      }
+    })
+
+    this.setState({nextTemplate: {...nextTemplate, values: nextValues}})
+  }
+
+  private handleUpdateTemplate = (nextNextTemplate: Template): void => {
+    const {nextTemplate} = this.state
+
+    const TemplateWithSelectedAndPicked = reconcileSelectedAndPickedValues(
```

Comment:
super weird to see `TemplateWithSelectedAndPicked` capitalized here and in every other instance... if it's not a type, is there a reason it's capitalized? otherwise, could you de-capitalize it here and in every other instance for consistency?